### DISCUSSION TITLE
how do i delete bot ownership via database?

### DISCUSSION THREAD

**Discussion by jacatinelord** [trickerer/Trinity-Bots#630] [Q&A] [answered]
Hello, I use npcbot with azeroth core and want to make the npcs expire within selected time, any other alternative? i want to use the command from console but it does delete the existence of the bot on the world.
I also tried to make it by the idea of deleting the ownership on database (characters_npcbot) and it doesnt work.

thank you!

**Comment by trickerer** [comment, marked as answer]
https://github.com/trickerer/Trinity-Bots/issues/579

Currently there is only one command which removes bot from a player and it requires selection, so it cannot be used from console.
DB changes only take effect at server startup so it isn't an ideal way to go about it.
